schema/snapcraft.yaml

```diff
@@ -88,6 +88,12 @@ properties:
           stop-command: ["daemon"]
         additionalProperties: false
         properties:
+          desktop:
+            type: string
+            description: path to a desktop file representing the app, relative to the prime directory
+          icon:
```

Comment:
is this here to support replacing icon names in existing desktop files instead of paths?